services/rsources/handler.go

```diff
@@ -404,3 +407,35 @@ func sqlFilters(jobRunId string, filter JobFilter) (fragment string, params []in
 	}
 	return filters, filterParams
 }
+
+func (sh *sourcesHandler) MonitorLagLoop(ctx context.Context) <-chan LagStatus {
+	lagCh := make(chan LagStatus)
+	logicalReplicationTrigger := func() <-chan time.Time {
+		return time.After(config.GetDuration("Rsources.stats.logRep.monitoring.interval", 10, time.Second))
```

Comment:
Can we shorten the config variable name?

Reply:
going with `Rsources.stats.logRepMonitoringInterval`